apps/web/src/app/organizations/billing/billing-sync-api-key.component.html

```diff
@@ -37,7 +37,7 @@ <h1 class="modal-title" id="billingSyncApiKeyTitle">
 
         <div *ngIf="clientSecret && !showRotateScreen">
           <p>{{ "copyPasteBillingSync" | i18n }}</p>
-          <label for="clientSecret">Billing Sync Key</label>
+          <label for="clientSecret">Billing sync token</label>
```

Comment:
should probably use the translated string.

```suggestion
          <label for="clientSecret">{{ "billingSyncKey" | i18n }}</label>
```

Reply:
Thanks for pointing this out! I checked the `messages.json` file and it looks like that translated string already has the correct text, so the original issue was that we had hardcoded that field label name. This is much better. 

`  "billingSyncKey": {
    "message": "Billing sync token"
  },`